x-pack/plugin/searchable-snapshots/src/main/plugin-metadata/plugin-security.codebases

```diff
@@ -0,0 +1 @@
+preallocate: org.elasticsearch.xpack.searchablesnapshots.preallocate.Preallocate
```

Comment:
This seems unrelated to this change?

Reply:
It's needed to be able to run the relevant unit tests inside of IntelliJ.